remove lya absorption below lyman-lymit

ngbusca opened the pull request:
This PR intends to address issue #235 by removing the lyman-alpha absorption below the lyman-limit (which is already taken into account in the template).

ngbusca commented:
@andreufont can you take a look?

andreufont commented:
@ngbusca, I'm not sure what you mean. In principle there is nothing wrong with having Lyman alpha forest down to the blue end of the spectrograph, right? Regardless of rest-frame wavelength.

ngbusca commented:
I've now found a different problem  at the red end when adding the forest described in #235 (bottom plots). I removed my previous (wrong) "fix"  and added a red-end fix. @moustakas I seem to be able to generate z~3.5 qsos with forest without a problem using this branch, can you double check?
